Jibus22/webserv
# Feature/cgi

features : 
 - directory listing
 - content-length header inserted, if not present, to cgi output

tests : 
 - add drunksite html/css files so we have an interface for testing
 - add bunch of cgi tests thru html forms, with python & perl scripts: GET & POST tested. document response & local/client redirection tested. (will have to check again the local redirection)
